Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -1624,5 +1624,25 @@ public void TestGetResultsOfUnknownQueryIdWithConfiguredRetry()
                 conn.Close();
             }
         }
+        
+        [Test]
+        public void TestSetQueryTagOverridesConnectionString()
```

Comment:
Imho both those tests (this new one and https://github.com/snowflakedb/snowflake-connector-net/blob/15f4c556ddb8d2487a8c1612a174be841ee35e63/Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs#L2228)
 are testing tagging a particular command in two scenarios 
and SFConnectionIT.TestConnectStringWithQueryTag belongs here as well.